src/js/ui/simply-pebble.js

```diff
@@ -1113,6 +1132,35 @@ SimplyPebble.accelConfig = function(def) {
   SimplyPebble.sendPacket(AccelConfigPacket.prop(def));
 };
 
+SimplyPebble.voiceDictationSession = function(callback) {
+  // If there's a transcription in progress
+  if (SimplyPebble.dictationCallback) {
+    callback( { 'status': -1, 'transcription': null } );
```

Comment:
No spaces for round parens, but the spaces for curly is good. I have to admit I'm not sure what you'd call this style. No quotes necessary, but you did use the right kind of quotes!
```
    callback({ status: -1, transcription: null });
```